seatunnel-engine/seatunnel-engine-storage/checkpoint-storage-plugins/checkpoint-storage-hdfs/src/main/java/org/apache/seatunnel/engine/checkpoint/storage/hdfs/HdfsStorageFactory.java

```diff
@@ -30,6 +30,25 @@
 
 import java.util.Map;
 
+/**
+ * HdfsCheckpointStorageFactory.
```

Comment:
Could you add a document about how to config checkpoint storage in `docs/en/seatunnel-engine` ?

Reply:
yup, I will add docs after this done